src/utils/index.ts

```diff
@@ -131,4 +131,11 @@ const handleDateTimeInput = (dateTimeValue: any) => {
   return DateTime.fromISO(dateTime).toMillis()
 }
 
-export { handleDateTimeInput, showToast, hasError , parseCsv , jsonToCsv, JsonToCsvOption, isFutureDate }
+const prepareRuntime = (job: any) => {
+  // For job frequency everyday, set to start of next day
+  if (job.frequency === 'EVERYDAY') {
+    return DateTime.now().startOf('day').toMillis() + 86400000;
```

Comment:
I think we should use plus method to add 1 day to it
https://moment.github.io/luxon/api-docs/index.html